Add docs for batch operations (#17283)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/syntax/batch-update.md

# BATCH UPDATE

{% note tip %}

Before diving into `BATCH UPDATE`, it is recommended to familiarize yourself with the standard [UPDATE](update.md).

{% endnote %}

> `BATCH UPDATE` allows to update records in large tables while minimizing the risk of lock invalidation and transaction rollback by weakening guarantees. Specifically, data updates are performed as a series of transactions for each [partition](../../../concepts/datamodel/table.md#partitioning) of the specified table separately, processing 10 000 rows per iteration. At the time of request execution, 10 partitions are processed simultaneously.

Reviewer: ```suggestion
`BATCH UPDATE` allows to update records in large tables while minimizing the risk of lock invalidation and transaction rollback by weakening guarantees. Specifically, data updates are performed as a series of transactions for each [partition](../../../concepts/datamodel/table.md#partitioning) of the specified table separately, processing 10 000 rows per iteration. Each query processes up to 10 partitions concurrently.
```

> `BATCH UPDATE` allows to update records in large tables, minimizing the risk of lock invalidation and transaction rollback by weakening guarantees, namely, data updates is performed as a series of transactions for each [partition](../../../concepts/datamodel/table.md#partitioning) of the specified table separately, processing a limited number of rows per iteration (by default, 10000 rows). At the time of request execution, a limited number of partitions are processed simultaneously (by default, no more than 10 partitions).

Reviewer: ```suggestion
`BATCH UPDATE` allows to update records in large tables while minimizing the risk of lock invalidation and transaction rollback by weakening guarantees. Specifically, data updates are performed as a series of transactions for each [partition](../../../concepts/datamodel/table.md#partitioning) of the specified table separately, processing a limited number of rows per iteration (by default, 10000 rows). At the time of request execution, a limited number of partitions are processed simultaneously (by default, no more than 10 partitions).
```
